state/migration_export.go

```diff
@@ -67,6 +67,7 @@ type ExportConfig struct {
 	SkipRelationData         bool
 	SkipInstanceData         bool
 	SkipApplicationOffers    bool
+	SkipOfferConnections     bool
```

Comment:
We should be setting these to true for bundle exports when we add new ones that it doesn't need.

Reply:
Ah, I hadn't thought about that!